eclipse/build

```diff
@@ -32,11 +32,11 @@ VSN=`get_feature_vsn org.erlide.kernel.feature`
 chmod u+x mvnw
 ./mvnw -B -U tycho-versions:set-version -DnewVersion=${VSN}.qualifier -Dartifacts=org.erlide.kernel.feature,org.erlide.kernel.debugger,org.erlide.kernel.ide,org.erlide.kernel.site,org.erlide.kernel.common
 # there will be an error if versions don't match
-git commit -am 'foo'
```

Comment:
Is the comment above referring to the git commit command? Perhaps there is a pre-commit hook that does some checks? Or should the comment be deleted?

Reply:
My understanding is that the comment belongs more to the line above,
where the `set-version` procedure updates all given artifact with the correct version...but I might be wrong.

I now see in the git history that the `git commit` line was added long after the comment, so I think the comment can stay for now.